Doc/library/operator.rst

```diff
@@ -17,9 +17,18 @@
 
 The :mod:`operator` module exports a set of efficient functions corresponding to
 the intrinsic operators of Python.  For example, ``operator.add(x, y)`` is
-equivalent to the expression ``x+y``.  The function names are those used for
+equivalent to the expression ``x+y``.  Some function names are those used for
 special class methods; variants without leading and trailing ``__`` are also
-provided for convenience.
+provided for backward compatibility.
+
+.. note::
+
+    The dunderless versions of the functions are preferred.
+
```

Comment:
Thank you for the constructive comment, it does make sense to not use `note`, because it denotes a certain emphasis.

I have updated it to be read as a paragraph.